src/ontology/iao-edit.owl

```diff
@@ -4013,6 +4014,22 @@ Data were captured into EPI-DATA (version 3.1), cleaned and then exported to Sta
         <obo:IAO_0000233>https://github.com/information-artifact-ontology/IAO/issues/259</obo:IAO_0000233>
         <rdfs:label xml:lang="en">ORCID identifier</rdfs:label>
     </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/IAO_0001000 -->
```

Comment:
Since you wrote that the formalization of the mapping this class to [DCAT:Dataset](https://www.w3.org/ns/dcat#Dataset) with SSSOM is outside of this PR, I was wondering if there speaks anything against adding a `skos:exactMatch DCAT:Dataset` annotation here already in this PR?